extension/js/content_scripts/webmail/setup-webmail-content-script.ts

```diff
@@ -329,6 +351,7 @@ export const contentScriptSetupIfVacant = async (webmailSpecific: WebmailSpecifi
       const clientConfiguration = await ClientConfiguration.newInstance(acctEmail);
       await startPullingKeysFromEkm(acctEmail, clientConfiguration, factory, ppEvent);
       await webmailSpecific.start(acctEmail, clientConfiguration, inject, notifications, factory, notifyMurdered);
+      await notifyExpiringKeys(acctEmail, clientConfiguration, notifications);
```

Comment:
Line 352 only starts pulling keys from EKM, the actual pull and processing are lines 295 and 296, so notification should be performed after line 296 (possibly in `catch` or `finally` block to properly handle a fetch error)

Reply:
In case `clientConfiguration.usesKeyManager()` is `false` then we can call it here